### ISSUE TITLE
NPE when clicking items in the hotbar while a Gui is open

### ISSUE THREAD

**Issue by cheracc** [TriumphTeam/triumph-gui#22]
I'm getting this on any type of Gui whenever I click on any item in the hotbar while it's open:
`[14:05:15 ERROR]: Could not pass event InventoryClickEvent to BattleGamesHungerRoyale v1.16.5_alpha1
java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because the return value of "dev.triumphteam.gui.components.util.ItemNbt.getString(org.bukkit.inventory.ItemStack, String)" is null
	at dev.triumphteam.gui.guis.GuiListener.isGuiItem(GuiListener.java:176) ~[?:?]
	at dev.triumphteam.gui.guis.GuiListener.onGuiClick(GuiListener.java:98) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor18.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-777]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-777]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-777]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2715) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1266) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1259) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1220) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1134) ~[patched_1.16.5.jar:git-Paper-777]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:291) ~[patched_1.16.5.jar:git-Paper-777]
	at java.lang.Thread.run(Thread.java:831) [?:?]
`

I don't really know how to do a PR or anything but in GuiListener.java:

```
private boolean isGuiItem(@Nullable final ItemStack currentItem, @Nullable final GuiItem guiItem) {
        if (currentItem == null || guiItem == null) return false;

        // Checks whether or not the Item is truly a GUI Item
        return ItemNbt.getString(currentItem, "mf-gui").equals(guiItem.getUuid().toString());

    }

```
ItemNbt.getString(currentItem, "mf-gui" needs a null check I think.

Thanks for the library I've been using it a lot.

**Comment by ipsk** [comment]
Which version are you using? That was fixed in the latest version b8832bcf379480ba9d52c6ca27986ca7db01467a.

**Comment by cheracc** [comment]
I noticed this awhile ago and just got around to reporting it - I see that I missed the boat.

All good, thank you!
